de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -390,12 +391,12 @@ public AddProjectToSessionWizard(Window parent,
         this.setPreferredSize(new Dimension(650,515));
 
 
-        List<ProjectNegotiationData> data = negotiation.getProjectNegotiationData();
+        List<ReferencePointNegotiationData> data = negotiation.getReferencePointNegotiationData();
 
         localProjects = new HashMap<String, IProject>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS2293))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq3SkEeD8QRkTSuw&open=AWcmJq3SkEeD8QRkTSuw)

[ISSUE_KEY]: AWcmJq3SkEeD8QRkTSuw